app/livechat/server/lib/screenSharing/index.ts

```diff
@@ -0,0 +1,12 @@
+// import { IScreenShareProvider } from './IScreenShareProvider';
+import { ScreenSharingManager } from '../ScreenSharingManager';
+import { CobrowseScreenShareProvider } from './CobrowseScreenShareProvider';
+
+
+// name of provider will come from settings
+export const ScreensharingManager = new ScreenSharingManager('cobrowse.io');
+
+// register all the proviers
+ScreensharingManager.registerProvider('cobrowse.io', new CobrowseScreenShareProvider());
+
+ScreensharingManager.setProvider();
```

Comment:
Why do you call the `setProvider` method here? Also, the method requires a parameter that you're not passing here.

Reply:
It does not require a parameter

My thought process was that 
(I) we will register all the providers in the index.ts 
 and we'll get which provider we are using from setting
(ii) const ScreensharingManager = new ScreenSharingManager('cobrowse.io');
     then I initialized the provider in the screenSharingManager from setting
(iii) ScreensharingManager.setProvider();
       the provider name is already set by constructor, therefore, it will set the provider